sidebars.js

```diff
@@ -343,14 +336,14 @@ module.exports = {
             { type: 'doc', id: 'reference/supporting_resources' }
           ]
         },
-        //{
-          //type: 'category',
-          //label: 'Spare Parts',
-          //collapsed: true,
-          //items: [
-            //{ type: 'doc', id: 'reference/spare_parts' }
-          //]
-        //},
+        // {
```

Comment:
Let's drop this entirely. There's no reason to keep the ancient Spare Parts section.

Reply:
Ok.. removing that commented section